hamilton/plugins/pandas_extensions.py

```diff
@@ -123,6 +125,266 @@ def load_data(self, type_: Type) -> Tuple[DATAFRAME_TYPE, Dict[str, Any]]:
     @classmethod
     def name(cls) -> str:
         return "csv"
+'''
```

Comment:
I'd just delete this old implementation.

Reply:
Corrected in most recent PR